build.assets/Dockerfile-centos7

```diff
@@ -68,16 +69,56 @@ RUN (groupadd ci --gid=$GID -o && useradd ci --uid=$UID --gid=$GID --create-home
      mkdir -p -m0700 /var/lib/teleport && chown -R ci /var/lib/teleport)
 
 RUN yum groupinstall -y 'Development Tools' && \
+    yum install -y epel-release && \
+    yum update -y && \
+    yum -y install centos-release-scl-rh && \
     yum install -y \
```

Comment:
I've tried and failed. You need to first install `centos-release-scl-rh` in order to be able to install `devtoolset-11-*`. Otherwise `yum` says that the package doesn't exist.